tests/Integration/GlobalRayLoaderTest.php

```diff
@@ -8,6 +8,14 @@
     expect(function_exists('ray'))->toBeTrue();
 });
 
-it('will not generate exceptions', function () {
+it('will resolve rd global function', function () {
+    expect(function_exists('rd'))->toBeTrue();
+});
+
+it('will not generate exceptions when calling ray', function () {
     ray('foo');
 })->expectNotToPerformAssertions();
+
+it('will not generate exceptions when calling rd', function () {
+    ray('rd');
```

Comment:
Is this correct, or should it be `rd()`?

Reply:
Ah great catch thanks @owenvoke! 🙏  It's for sure supposed to be `rd()`

Reply:
Oh, I forgot -- we can't test `rd()` because it kills the test script 😅

Reply:
Oh, so it's like `ray` and then `die`? Or something?

Reply:
Yea that's exactly what it is 👍